src/QsCompiler/Core/SymbolTable/PartialNamespace.fs

```diff
@@ -23,7 +23,7 @@ type private PartialNamespace
         name: string,
         source: string,
         documentation: IEnumerable<ImmutableArray<string>>,
-        openNS: IEnumerable<KeyValuePair<string, string>>,
+        openNS: IEnumerable<KeyValuePair<string, HashSet<string>>>,
```

Comment:
Unfortunately, it looks like this change also affects the public API surface... this line in IQ# expects to be run on a single string but won't work properly against a hashset: https://github.com/microsoft/iqsharp/blob/cad319493cd00934a40e1c37e2856fa18d3a1543/src/Core/Compiler/CompilerService.cs#L119
Since this is pretty fundamental to the feature, I think we'll need to keep this one here and make it a cross-repo change coordinated with IQ#.

Reply:
Actually, never mind... I have the wrong spot in the code. I'll close this comment and put one on the right spot.